.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -16,5 +16,6 @@
     <inspection_tool class="PreviewNeedsComposableAnnotation" enabled="true" level="ERROR" enabled_by_default="true">
       <option name="previewFile" value="true" />
     </inspection_tool>
+    <inspection_tool class="Reformat" enabled="true" level="WEAK WARNING" enabled_by_default="true" />
```

Comment:
Adds the ability for the project to be reformatted according to the project settings.  Add per the configure style in IDE section here: https://kotlinlang.org/docs/coding-conventions.html#configure-style-in-ide